edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Kubernetes.Test/commands/KubernetesCrdCommandTest.cs

```diff
@@ -111,6 +118,8 @@ public async void CrdCommandExecuteWithAuthCreateNewObjects()
                 Assert.True(postSecretCalled, nameof(postSecretCalled));
                 Assert.True(getCrdCalled, nameof(getCrdCalled));
                 Assert.True(postCrdCalled, nameof(postCrdCalled));
+                Assert.True(postedEdgeDeploymentDefinition.Spec[0].Env.Contains(new KeyValuePair<string,EnvVal>( "ACamelCaseEnvVar", new EnvVal("ACamelCaseEnvVarValue"))));
```

Comment:
This test started to verify too many things. It checks that both api method called as well as data serialized correctly. Should we move data checks to separated test?